{
  "stats": {
    "suites": 1,
    "tests": 4,
    "passes": 0,
    "pending": 0,
    "failures": 4,
    "start": "2025-06-08T05:54:05.693Z",
    "end": "2025-06-08T05:54:09.084Z",
    "duration": 3391,
    "testsRegistered": 4,
    "passPercent": 0,
    "pendingPercent": 0,
    "other": 0,
    "hasOther": false,
    "skipped": 0,
    "hasSkipped": false
  },
  "results": [
    {
      "uuid": "cd5532b7-4f74-48d1-a6b8-ea9c6f7748e1",
      "title": "",
      "fullFile": "cypress\\e2e\\api_test.cy.js",
      "file": "cypress\\e2e\\api_test.cy.js",
      "beforeHooks": [],
      "afterHooks": [],
      "tests": [],
      "suites": [
        {
          "uuid": "d828bbc9-eb47-410a-bb64-a576b7696035",
          "title": "Dummy api test cases",
          "fullFile": "",
          "file": "",
          "beforeHooks": [],
          "afterHooks": [],
          "tests": [
            {
              "title": "Get -fetch list of Posts ",
              "fullTitle": "Dummy api test cases Get -fetch list of Posts ",
              "timedOut": null,
              "duration": 880,
              "state": "failed",
              "speed": null,
              "pass": false,
              "fail": true,
              "pending": false,
              "context": null,
              "code": "cy.request(`${baseUrl}/posts`).then(response => {\n  expect(response.status).to.eq(200);\n  expect(response.body).to.have.length(100);\n  cy.log(response.body);\n});",
              "err": {
                "message": "CypressError: `cy.request()` failed trying to load:\n\nhttp://localhost:3000/APIBatchStudents/posts\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > AggregateError\n\n-----------------------------------------------------------\n\nThe request we sent was:\n\nMethod: GET\nURL: http://localhost:3000/APIBatchStudents/posts\n\n-----------------------------------------------------------\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n\nhttps://on.cypress.io/request",
                "estack": "CypressError: `cy.request()` failed trying to load:\n\nhttp://localhost:3000/APIBatchStudents/posts\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > AggregateError\n\n-----------------------------------------------------------\n\nThe request we sent was:\n\nMethod: GET\nURL: http://localhost:3000/APIBatchStudents/posts\n\n-----------------------------------------------------------\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n\nhttps://on.cypress.io/request\n    at <unknown> (http://localhost:50932/__cypress/runner/cypress_runner.js:135007:70)\n    at tryCatcher (http://localhost:50932/__cypress/runner/cypress_runner.js:1777:23)\n    at <unknown> (http://localhost:50932/__cypress/runner/cypress_runner.js:4173:37)\n    at tryCatcher (http://localhost:50932/__cypress/runner/cypress_runner.js:1777:23)\n    at Promise._settlePromiseFromHandler (http://localhost:50932/__cypress/runner/cypress_runner.js:1489:31)\n    at Promise._settlePromise (http://localhost:50932/__cypress/runner/cypress_runner.js:1546:18)\n    at Promise._settlePromise0 (http://localhost:50932/__cypress/runner/cypress_runner.js:1591:10)\n    at Promise._settlePromises (http://localhost:50932/__cypress/runner/cypress_runner.js:1667:18)\n    at _drainQueueStep (http://localhost:50932/__cypress/runner/cypress_runner.js:2377:12)\n    at _drainQueue (http://localhost:50932/__cypress/runner/cypress_runner.js:2370:9)\n    at Async._drainQueues (http://localhost:50932/__cypress/runner/cypress_runner.js:2386:5)\n    at Async.drainQueues (http://localhost:50932/__cypress/runner/cypress_runner.js:2256:14)\nFrom Your Spec Code:\n    at Context.eval (webpack://cypressautomation/./cypress/e2e/api_test.cy.js:5:7)\n\nFrom Node.js Internals:\n  RequestError: AggregateError\n      at new r (<embedded>:227:441488)\n      at ee.callback (<embedded>:227:447489)\n      at e.callback.s.callback [as _callback] (<embedded>:227:446935)\n      at s._callback.s.callback.s.callback (<embedded>:290:105750)\n      at ee.emit (node:events:518:28)\n      at ee.onRequestError (<embedded>:290:114692)\n      at ClientRequest.emit (node:events:530:35)\n      at emitErrorEvent (node:_http_client:101:11)\n      at Socket.socketErrorListener (node:_http_client:504:5)\n      at Socket.emit (node:events:518:28)\n      at emitErrorNT (node:internal/streams/destroy:169:8)\n      at emitErrorCloseNT (node:internal/streams/destroy:128:3)\n      at process.processTicksAndRejections (node:internal/process/task_queues:82:21)",
                "diff": null
              },
              "uuid": "f78457bf-f6a3-438b-96e7-7e0d0b4a726f",
              "parentUUID": "d828bbc9-eb47-410a-bb64-a576b7696035",
              "isHook": false,
              "skipped": false
            },
            {
              "title": "Post -Create a new record ",
              "fullTitle": "Dummy api test cases Post -Create a new record ",
              "timedOut": null,
              "duration": 881,
              "state": "failed",
              "speed": null,
              "pass": false,
              "fail": true,
              "pending": false,
              "context": null,
              "code": "cy.request('POST', `${baseUrl}/posts`, {\n  title: 'foo',\n  body: 'bar',\n  userId: 1\n}).then(response => {\n  expect(response.status).to.eq(201);\n  expect(response.body).to.have.property('id');\n});",
              "err": {
                "message": "CypressError: `cy.request()` failed trying to load:\n\nhttp://localhost:3000/APIBatchStudents/posts\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > AggregateError\n\n-----------------------------------------------------------\n\nThe request we sent was:\n\nMethod: POST\nURL: http://localhost:3000/APIBatchStudents/posts\n\n-----------------------------------------------------------\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n\nhttps://on.cypress.io/request",
                "estack": "CypressError: `cy.request()` failed trying to load:\n\nhttp://localhost:3000/APIBatchStudents/posts\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > AggregateError\n\n-----------------------------------------------------------\n\nThe request we sent was:\n\nMethod: POST\nURL: http://localhost:3000/APIBatchStudents/posts\n\n-----------------------------------------------------------\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n\nhttps://on.cypress.io/request\n    at <unknown> (http://localhost:50932/__cypress/runner/cypress_runner.js:135007:70)\n    at tryCatcher (http://localhost:50932/__cypress/runner/cypress_runner.js:1777:23)\n    at <unknown> (http://localhost:50932/__cypress/runner/cypress_runner.js:4173:37)\n    at tryCatcher (http://localhost:50932/__cypress/runner/cypress_runner.js:1777:23)\n    at Promise._settlePromiseFromHandler (http://localhost:50932/__cypress/runner/cypress_runner.js:1489:31)\n    at Promise._settlePromise (http://localhost:50932/__cypress/runner/cypress_runner.js:1546:18)\n    at Promise._settlePromise0 (http://localhost:50932/__cypress/runner/cypress_runner.js:1591:10)\n    at Promise._settlePromises (http://localhost:50932/__cypress/runner/cypress_runner.js:1667:18)\n    at _drainQueueStep (http://localhost:50932/__cypress/runner/cypress_runner.js:2377:12)\n    at _drainQueue (http://localhost:50932/__cypress/runner/cypress_runner.js:2370:9)\n    at Async._drainQueues (http://localhost:50932/__cypress/runner/cypress_runner.js:2386:5)\n    at Async.drainQueues (http://localhost:50932/__cypress/runner/cypress_runner.js:2256:14)\nFrom Your Spec Code:\n    at Context.eval (webpack://cypressautomation/./cypress/e2e/api_test.cy.js:14:11)\n\nFrom Node.js Internals:\n  RequestError: AggregateError\n      at new r (<embedded>:227:441488)\n      at ee.callback (<embedded>:227:447489)\n      at e.callback.s.callback [as _callback] (<embedded>:227:446935)\n      at s._callback.s.callback.s.callback (<embedded>:290:105750)\n      at ee.emit (node:events:518:28)\n      at ee.onRequestError (<embedded>:290:114692)\n      at ClientRequest.emit (node:events:530:35)\n      at emitErrorEvent (node:_http_client:101:11)\n      at Socket.socketErrorListener (node:_http_client:504:5)\n      at Socket.emit (node:events:518:28)\n      at emitErrorNT (node:internal/streams/destroy:169:8)\n      at emitErrorCloseNT (node:internal/streams/destroy:128:3)\n      at process.processTicksAndRejections (node:internal/process/task_queues:82:21)",
                "diff": null
              },
              "uuid": "426cc31d-4c5c-4e30-a4d6-80b8e4d03215",
              "parentUUID": "d828bbc9-eb47-410a-bb64-a576b7696035",
              "isHook": false,
              "skipped": false
            },
            {
              "title": "Put -Update existing record ",
              "fullTitle": "Dummy api test cases Put -Update existing record ",
              "timedOut": null,
              "duration": 715,
              "state": "failed",
              "speed": null,
              "pass": false,
              "fail": true,
              "pending": false,
              "context": null,
              "code": "cy.request('PUT', `${baseUrl}/posts/1`, {\n  id: 1,\n  title: 'updated title',\n  body: 'updated body',\n  userId: 1\n}).then(response => {\n  expect(response.status).to.eq(200);\n  expect(response.body.title).to.eq('updated title');\n});",
              "err": {
                "message": "CypressError: `cy.request()` failed trying to load:\n\nhttp://localhost:3000/APIBatchStudents/posts/1\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > AggregateError\n\n-----------------------------------------------------------\n\nThe request we sent was:\n\nMethod: PUT\nURL: http://localhost:3000/APIBatchStudents/posts/1\n\n-----------------------------------------------------------\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n\nhttps://on.cypress.io/request",
                "estack": "CypressError: `cy.request()` failed trying to load:\n\nhttp://localhost:3000/APIBatchStudents/posts/1\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > AggregateError\n\n-----------------------------------------------------------\n\nThe request we sent was:\n\nMethod: PUT\nURL: http://localhost:3000/APIBatchStudents/posts/1\n\n-----------------------------------------------------------\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n\nhttps://on.cypress.io/request\n    at <unknown> (http://localhost:50932/__cypress/runner/cypress_runner.js:135007:70)\n    at tryCatcher (http://localhost:50932/__cypress/runner/cypress_runner.js:1777:23)\n    at <unknown> (http://localhost:50932/__cypress/runner/cypress_runner.js:4173:37)\n    at tryCatcher (http://localhost:50932/__cypress/runner/cypress_runner.js:1777:23)\n    at Promise._settlePromiseFromHandler (http://localhost:50932/__cypress/runner/cypress_runner.js:1489:31)\n    at Promise._settlePromise (http://localhost:50932/__cypress/runner/cypress_runner.js:1546:18)\n    at Promise._settlePromise0 (http://localhost:50932/__cypress/runner/cypress_runner.js:1591:10)\n    at Promise._settlePromises (http://localhost:50932/__cypress/runner/cypress_runner.js:1667:18)\n    at _drainQueueStep (http://localhost:50932/__cypress/runner/cypress_runner.js:2377:12)\n    at _drainQueue (http://localhost:50932/__cypress/runner/cypress_runner.js:2370:9)\n    at Async._drainQueues (http://localhost:50932/__cypress/runner/cypress_runner.js:2386:5)\n    at Async.drainQueues (http://localhost:50932/__cypress/runner/cypress_runner.js:2256:14)\nFrom Your Spec Code:\n    at Context.eval (webpack://cypressautomation/./cypress/e2e/api_test.cy.js:30:11)\n\nFrom Node.js Internals:\n  RequestError: AggregateError\n      at new r (<embedded>:227:441488)\n      at ee.callback (<embedded>:227:447489)\n      at e.callback.s.callback [as _callback] (<embedded>:227:446935)\n      at s._callback.s.callback.s.callback (<embedded>:290:105750)\n      at ee.emit (node:events:518:28)\n      at ee.onRequestError (<embedded>:290:114692)\n      at ClientRequest.emit (node:events:530:35)\n      at emitErrorEvent (node:_http_client:101:11)\n      at Socket.socketErrorListener (node:_http_client:504:5)\n      at Socket.emit (node:events:518:28)\n      at emitErrorNT (node:internal/streams/destroy:169:8)\n      at emitErrorCloseNT (node:internal/streams/destroy:128:3)\n      at process.processTicksAndRejections (node:internal/process/task_queues:82:21)",
                "diff": null
              },
              "uuid": "eda40672-ff6a-482e-aa19-997f6adc8d7a",
              "parentUUID": "d828bbc9-eb47-410a-bb64-a576b7696035",
              "isHook": false,
              "skipped": false
            },
            {
              "title": "Delete - existing record ",
              "fullTitle": "Dummy api test cases Delete - existing record ",
              "timedOut": null,
              "duration": 712,
              "state": "failed",
              "speed": null,
              "pass": false,
              "fail": true,
              "pending": false,
              "context": null,
              "code": "cy.request('DELETE', `${baseUrl}/myy`).then(response => {\n  expect(response.status).to.eq(200);\n});",
              "err": {
                "message": "CypressError: `cy.request()` failed trying to load:\n\nhttp://localhost:3000/APIBatchStudents/myy\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > AggregateError\n\n-----------------------------------------------------------\n\nThe request we sent was:\n\nMethod: DELETE\nURL: http://localhost:3000/APIBatchStudents/myy\n\n-----------------------------------------------------------\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n\nhttps://on.cypress.io/request",
                "estack": "CypressError: `cy.request()` failed trying to load:\n\nhttp://localhost:3000/APIBatchStudents/myy\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > AggregateError\n\n-----------------------------------------------------------\n\nThe request we sent was:\n\nMethod: DELETE\nURL: http://localhost:3000/APIBatchStudents/myy\n\n-----------------------------------------------------------\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n\nhttps://on.cypress.io/request\n    at <unknown> (http://localhost:50932/__cypress/runner/cypress_runner.js:135007:70)\n    at tryCatcher (http://localhost:50932/__cypress/runner/cypress_runner.js:1777:23)\n    at <unknown> (http://localhost:50932/__cypress/runner/cypress_runner.js:4173:37)\n    at tryCatcher (http://localhost:50932/__cypress/runner/cypress_runner.js:1777:23)\n    at Promise._settlePromiseFromHandler (http://localhost:50932/__cypress/runner/cypress_runner.js:1489:31)\n    at Promise._settlePromise (http://localhost:50932/__cypress/runner/cypress_runner.js:1546:18)\n    at Promise._settlePromise0 (http://localhost:50932/__cypress/runner/cypress_runner.js:1591:10)\n    at Promise._settlePromises (http://localhost:50932/__cypress/runner/cypress_runner.js:1667:18)\n    at _drainQueueStep (http://localhost:50932/__cypress/runner/cypress_runner.js:2377:12)\n    at _drainQueue (http://localhost:50932/__cypress/runner/cypress_runner.js:2370:9)\n    at Async._drainQueues (http://localhost:50932/__cypress/runner/cypress_runner.js:2386:5)\n    at Async.drainQueues (http://localhost:50932/__cypress/runner/cypress_runner.js:2256:14)\nFrom Your Spec Code:\n    at Context.eval (webpack://cypressautomation/./cypress/e2e/api_test.cy.js:45:11)\n\nFrom Node.js Internals:\n  RequestError: AggregateError\n      at new r (<embedded>:227:441488)\n      at ee.callback (<embedded>:227:447489)\n      at e.callback.s.callback [as _callback] (<embedded>:227:446935)\n      at s._callback.s.callback.s.callback (<embedded>:290:105750)\n      at ee.emit (node:events:518:28)\n      at ee.onRequestError (<embedded>:290:114692)\n      at ClientRequest.emit (node:events:530:35)\n      at emitErrorEvent (node:_http_client:101:11)\n      at Socket.socketErrorListener (node:_http_client:504:5)\n      at Socket.emit (node:events:518:28)\n      at emitErrorNT (node:internal/streams/destroy:169:8)\n      at emitErrorCloseNT (node:internal/streams/destroy:128:3)\n      at process.processTicksAndRejections (node:internal/process/task_queues:82:21)",
                "diff": null
              },
              "uuid": "a1bfb450-406e-4ada-aabf-8834ae7d1e2e",
              "parentUUID": "d828bbc9-eb47-410a-bb64-a576b7696035",
              "isHook": false,
              "skipped": false
            }
          ],
          "suites": [],
          "passes": [],
          "failures": [
            "f78457bf-f6a3-438b-96e7-7e0d0b4a726f",
            "426cc31d-4c5c-4e30-a4d6-80b8e4d03215",
            "eda40672-ff6a-482e-aa19-997f6adc8d7a",
            "a1bfb450-406e-4ada-aabf-8834ae7d1e2e"
          ],
          "pending": [],
          "skipped": [],
          "duration": 3188,
          "root": false,
          "rootEmpty": false,
          "_timeout": 2000
        }
      ],
      "passes": [],
      "failures": [],
      "pending": [],
      "skipped": [],
      "duration": 0,
      "root": true,
      "rootEmpty": true,
      "_timeout": 2000
    }
  ],
  "meta": {
    "mocha": {
      "version": "7.2.0"
    },
    "mochawesome": {
      "options": {
        "quiet": false,
        "reportFilename": "mochawesome",
        "saveHtml": false,
        "saveJson": true,
        "consoleReporter": "spec",
        "useInlineDiffs": false,
        "code": true
      },
      "version": "7.1.3"
    },
    "marge": {
      "options": {
        "reportDir": "cypress/reports",
        "overwrite": false,
        "html": false,
        "json": true
      },
      "version": "6.2.0"
    }
  }
}